capz/templates/gmsa-ci.yaml

```diff
@@ -83,6 +83,11 @@ spec:
           sc.exe start sshd
         path: C:/collect-hns-crashes.ps1
         permissions: "0744"
+      - content: |
+          ${AZURE_SSH_PUBLIC_KEY:=""}
+        owner: root:root
+        path: $env:ProgramData\ssh\administrators_authorized_keys
+        permissions: "0644"
```

Comment:
nit: the `icacls.exe` in the docs linked in the PR description only grant permissions to admin and system accounts, it doesn't grant read access to users. Should we update this to 0640?

Reply:
Good catch, let me dig into it

Reply:
Tested and 0640 also works and pushed the change. It doesn't seem like the permissions have any impact on the file though, I'll explore further as I dig into the test failures